slugid/slugid_test.go

```diff
@@ -12,21 +11,23 @@ import (
 func TestSlugidV4(t *testing.T) {
 	assert := assert.New(t)
 
-	slug, _ := v4()
-	match, err := regexp.MatchString("^[A-Za-z0-9-_]{22}$", slug)
-
-	assert.Nil(err, fmt.Sprintf("Error matching string generated by v4(): %s", err))
-	assert.Equal(match, true, fmt.Sprintf("Slug generated by v4() is invalid."))
+	// Let's test 1000 generated slugs, to increase confidence
+	for i := 0; i < 1000; i++ {
+		slug, _ := v4()
+		match := V4_SLUG_REGEXP.MatchString(slug)
+		assert.Equal(match, true, fmt.Sprintf("Slug %q generated by v4() is invalid.", slug))
```

Comment:
[`assert.True(match, ...)`](https://godoc.org/github.com/stretchr/testify/require#Assertions.True) can be used instead of `assert.Equal(match, true, ...)`.